src/pages/AccountRequest/accountUtil.tsx

```diff
@@ -0,0 +1,55 @@
+import { IFormOption } from "components/Form/interfaces";
+
+export interface IUserFormValues {
+  username: string;
+  email: string;
+  full_name: string;
+  role_id: number;
+  introduction: string;
+  institution_id: number;
+}
+
+export enum EmailPreference {
+  EMAIL_ON_REVIEW = "email_on_review",
+  EMAIL_ON_SUBMISSION = "email_on_submission",
+  EMAIL_ON_META_REVIEW = "email_on_review_of_review",
+}
+
+export interface IRole {
```

Comment:
Consider utilizing the `IRole` and `IInstitution` interfaces from `utils/interfaces.ts` as they are used in various locations throughout the code.